charter.html

```diff
@@ -178,24 +179,54 @@ <h3 id=chair-selection>Chair Selection</h3>
 <a href=https://www.w3.org/People/functions/w3m#dom>Dominique
 Hazaël-Massieux</a>.
 
-<h2 id=deliverables>Deliverables</h2>
+<h2 id=proposals>Proposals</h2>
+
+<p>A <dfn>Proposal</dfn> is an idea brought to the Community Group for
+consideration and potential adoption as a <a href=#work-items>Work
+Item</a>.
+
+<p>Any Community Group Participant may make a Proposal by filing
+<a href=https://github.com/privacycg/proposals/issues>an issue</a> in
+the
+<a href=https://github.com/privacycg/proposals><code>privacycg/proposals</code></a>
+repository on GitHub, stating the problem they would like to address and
+how they propose to address it.
+
+<p>The Community Group may discuss the Proposal on GitHub and during
+teleconferences or face-to-face meetings.
+
+<p>The <a href=#chairs>Chairs</a> may create a dedicated repository for
+a Proposal. They may do this for any reason, such as but not limited to:
+
+<ul>
+<li>because the Proposal has become sufficiently complex that writing an
+explainer is warranted
+<li>because the Proposal is sufficiently complex that having a dedicated
+issue tracker for it is warranted
+<li>because multiple <a href=#implementer>implementers</a> have
+expressed interest in the Proposal
+</ul>
 
-<p>The <a href=#chairs>Chairs</a> may add Deliverables, but must not add
-Deliverables which lack the support of at least
-two <a href=#implementer>implementers</a>.
+<p>Proposals may be withdrawn by their originators, or may be closed by
+the Chairs (if, for example, they fail to gain
+sufficient <a href=#implementer>implementer</a> support to be adopted as
+a Work Item).
 
-<p>The Chairs may remove Deliverables.
+<h2 id=work-items>Work Items</h2>
```

Comment:
Are these "Draft Community Group Reports" in the terminology of https://www.w3.org/community/about/agreements/#cgroups?

Reply:
yes, CG-DRAFT in Bikeshed-ese.

Reply:
"Draft Community Group Reports" is much wordier than "Work Item", though. I mean, "Work Item" is already too wordy.

Reply:
(and Proposals are not Draft Community Group Reports, they are just ideas someone had that may or may not go anywhere.)